libs/barista-components/drawer/BUILD.bazel

```diff
@@ -16,7 +46,49 @@ filegroup(
     ]),
 )
 
+multi_sass_binary(
```

Comment:
While searching for documentation on `multi_sass_binary`, I stumbled upon this:

> WARNING: This rule does a global compilation, and thus any change in the sources will trigger a build for all files. It is inefficient. Always prefer sass_binary and provide strict dependencies for most efficient compilation. This rule is also not used internally at Google.

Just wanted to point this out in case you aren't aware of it, though I guess you can't use `sass_binary` and `sass_library` anyway in this case.

Reply:
I know this but If I change something in the drawer I have to compile all 3 sass files :)

This is more on an app-level that you split up your scss files in multiple binaries and not use one multi sass for all.